src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -143,6 +143,7 @@ public SortedSet(IEnumerable<T> collection, IComparer<T> comparer)
                 {
                     Array.Sort(els, 0, count, _comparer);
```

Comment:
For consistency (always reading the local), I'd suggest moving your line to before this one, and changing this to pass the normalized local (vs reading the field).

Heck, for maximum consistency it seems like you want to replace the local/argument at the first line of this method, and then just use that consistently.